breadbox/breadbox/crud/dataset_reference.py

```diff
@@ -2,54 +2,6 @@
 
 from breadbox.db.session import SessionWithUser
 from ..models.dataset import (
-    DatasetReference,
     DimensionType,
     TabularDataset,
 )
```

Comment:
Could this file just be deleted now? It looks like the imports are the only part that haven't already been removed.

Reply:
Yes, good point. I'll delete.